modules/client/src/client-common/utils.ts

```diff
@@ -71,7 +71,13 @@ export function computeProposalStatusFilter(
   return where;
 }
 
-export function isProposalId(propoosalId: string): boolean {
+export function isProposalId(proposalId: string): boolean {
   const regex = new RegExp(/^0x[A-Fa-f0-9]{40}_0x[A-Fa-f0-9]{1,}$/i);
-  return regex.test(propoosalId);
+  return regex.test(proposalId);
+}
+export function isIpfsUri(ipfsUri: string): boolean {
+  const regex = new RegExp(
```

Comment:
IPFS URI's can theoretically have a path after the CID. We are not supporting it at this point, but `isIpfsUri` sounds bad if it doesn't support paths

Reply:
Ok, i'' edit the regex to accept paths :+1: